### DISCUSSION TITLE
Migrating from `requests`: `stream=True` in `Client.request`

### DISCUSSION THREAD

**Discussion by jamesbraza** [encode/httpx#3219] [Q&A] [answered]
I am working with a Google library that uses `transport.request(..., **kwargs)` with a kwarg of `stream=True`: https://github.com/googleapis/google-resumable-media-python/blob/v2.7.0/google/resumable_media/requests/download.py#L210

This library was designed to work with `requests`. I am trying to use `httpx.Client` for this, and am getting: `TypeError: httpx.Client.request() got an unexpected keyword argument 'stream'`

I see in the docs: https://www.python-httpx.org/compatibility/#streaming-responses
That the `httpx` interface is `with client.stream(...)`.

How can I get `httpx.Client.request` to work with `stream=True` keyword argument?

**Comment by kchen** [comment, marked as answer]
While I was searching the issues section previously, I had found #830, which indicates that the way to do this if you can't use with client.stream() is:

> client = httpx.Client()
> request = client.build_request("GET", url)
> response = client.send(request, stream=True)

and then making sure to call response.close() when ready, since that will not be automatically handled.
